google/api/annotations.proto

```diff
@@ -69,21 +80,16 @@ extend google.protobuf.FieldOptions {
   // See `google/api/resources.proto`.
   string resource_type = 1055;
```

Comment:
"resource_type" should be "resource_reference"

Reply:
_Ibid._

(Resolving this comment, and will handle both based on what is decided above.)